Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -288,16 +288,19 @@ struct TranslateResult
 };
 TranslateResult JitCache_TranslateAddress(u32 address);
 
-static const int BAT_INDEX_SHIFT = 17;
+constexpr int BAT_INDEX_SHIFT = 17;
```

Comment:
Considering the usage of this constant, wouldn't be preferable to also have an extra constant for ```1 << BAT_INDEX_SHIFT``` and functions for left and right shifting operations?